evergreen.yaml

```diff
@@ -167,67 +167,50 @@ task_groups:
 #           Buildvariants             #
 #######################################
 buildvariants:
-  - name: race-detector
```

Comment:
I would strongly recommend keeping the race detector because Jasper is a heavily race-sensitive repo due to its usage of goroutines.

Reply:
It has been incorporated into the standard Ubuntu 22.04 with `RACE_DETECTOR: true`. Interestingly it caught a race that it hadn't before, hence all my changes with pipeReader.